### ISSUE TITLE
Mouse scroll zoom scrolls also the page content

### ISSUE THREAD

**Issue by johnvarv** [chartjs/Chart.Zoom.js#29]
After the Chart.js 2.2.1 update the zoom functionality with the mouse scroll also scrolls the page. 

Noticed also in the **zoom.html** of your samples directory. I can reproduce it in a jsbin if you prefer.

Also I think I found a way to prevent this in your code. If you confirm the bug I can prepare a PR.

**Comment by etimberg** [comment]
this sounds like a bug

**Comment by icrm-amusienko** [comment]
@johnvarv, could you share the code how to fix the issue?
I want to fix in my local file before official release will be provided.

**Comment by johnvarv** [comment]
@icrm-amusienko Sorry for the super late response but unfortunately I was off the grid for a while now. I noticed there was a new release (0.2.0) that I missed since I did not prepare the PR in time. Anyway, the fix is in the *306* line of the dist file **Chart.zoom.js** :

```javascript
var wheelHandler = function(e) {
				var rect = e.target.getBoundingClientRect();
				var offsetX = e.clientX - rect.left;
				var offsetY = e.clientY - rect.top;
	
				var center = {
					x : offsetX,
					y : offsetY
				};
	
				if (e.deltaY < 0) {
					doZoom(chartInstance, 1.1, center);
				} else {
					doZoom(chartInstance, 0.909, center);
				}
			    e.preventDefault();
			};
```

The change is actually to insert the **e.preventDefault()** in order to prevent the mouse wheel to scroll.

I will try to prepare a PR as I promised but my schedule is really tight, apologies for that.

**Comment by johnvarv** [comment]
@etimberg I issued a PR [#36](https://github.com/chartjs/Chart.Zoom.js/pull/36) . I couldn't find a **test** gulp task to run though. I also noticed that there was an older PR [#24](https://github.com/chartjs/Chart.Zoom.js/pull/24) with the same content but somehow after a release was not included in the dist.

**Comment by icrm-amusienko** [comment]
@johnvarv, thanks a lot!

**Comment by etimberg** [comment]
Closed in #36
